src/code_80A2C3C.c

```diff
@@ -0,0 +1,34 @@
+#include "global.h"
+#include "bmsave.h"
+
+extern u8 LoadAndVerifySecureHeaderSW(struct SecureSaveHeader* buf);
+extern void SaveSecureHeader(struct SecureSaveHeader* header);
+
+// unitID is CharacterData.number. Should I just use "number" as variable name?
```

Comment:
use character id/`charId` here ("unit" and "character" should refer to different things and this is a character id). (I use PID in fe6 for this).